### PR TITLE
Added support for Gruntfile.coffee

### PR DESCRIPTION
**Pull request by skanne**
Hi!

While using this atom-build plugin in Atom I've noticed that my Grunt tasks weren't listed even when refreshed once I switched/converted from Gruntfile.js to Gruntfile.coffee. So I decided to fork your repository and added the support thereof. Hope you accept this pull request.

Best regards,
Sven

### CODE REVIEW

File: lib/parser-task.js [hunk 1]
```diff
@@ -25,7 +25,11 @@ module.exports = function (path) {
     window = undefined;
     /* eslint-enable no-native-reassign, no-undef */
 
-    require(path + '/Gruntfile.js')(grunt);
+    if (fs.existsSync(path + '/Gruntfile.coffee')) {
+      require(path + '/Gruntfile.coffee')(grunt);
+    } else {
+      require(path + '/Gruntfile.js')(grunt);
+    }
```
Comment (noseglid) [inline lib/parser-task.js:34]: `fs` is not defined. You must require it. Also, are you sure it is possible to `require` a `coffee` file?

**Comment by noseglid** [comment]
I would also like to see some tests where targets are extracted from a `Gruntfile.coffee`. You can look at other tests for inspiration.

**Comment by skanne** [comment]
Ok, sorry, did it in a night session where I actually should have gone to bed long before instead of submitting half-baked pull requests. I've fixed the missing `fs` import and created test cases. And yes, you can `require` a `coffee` file. That's also what [atom-build-gulp](https://github.com/AtomBuild/atom-build-gulp) does. See [here](https://github.com/AtomBuild/atom-build-gulp/blob/master/lib/gulp.js#L28) and [here](https://github.com/AtomBuild/atom-build-gulp/blob/master/lib/parser-task.js#L20). Right now I can't find the link(s) saying that you can `require` coffee files as well. But I've tested it in Atom 1.5.3 and it works perfectly.

**Comment by noseglid** [comment]
Okay cool! There's some [linting errors](https://travis-ci.org/AtomBuild/atom-build-grunt/builds/111975560) though, then I'd be happy to merge this!

**Comment by skanne** [comment]
Sådär, nu är den också fixad. Tack för allt! Ha en bra dag. ;-)

**Comment by noseglid** [comment]
Thanks for this! :)

**Pull request merged**